### PR TITLE
Fix style of popups

### PR DESCRIPTION
**Pull request by maov19**
Dear code reviewer, 

Please consider this PR as a continuation of [Popups in Projects section #12,](https://github.com/maov19/my_portfolio/pull/12)  
- By omission, the branch was merged with the `form_validation` branch
- `form_validation` was merged to main earlier today in [PR 13](https://github.com/maov19/my_portfolio/pull/13).

I wasn't able to unmerge or reopen PR#12, but since its still _alive_, I created this PR#15 on the same branch.
You can check the merge on the commit `a2d5f1acb3338b5fa9489cd46e5f4c4ddf3d2990` (it took me a while to figure it out)

### CODE REVIEW

**Comment by maov19** [comment]
**Reviewer's  last comment:**

`You seem to have introduced a bug as a result of the changes you made. It seems like the image takes a lot of the space so I cant view the description section and the links button. Please fix this`
- The issue was fixed by adding a little y-axis overflow. 
_The **Details** figma design has 100% width images, and its height its a little larger than the **Main-scroll** viewport, hence the overflow_ 

image

File: index.html [hunk 1]
```diff
@@ -65,6 +63,8 @@ <h2 class="presentation">
 
     <!-- works -->
     <section id="portfolio" class="works">
```
Comment (1GORDON) [inline index.html:65]: Great job @maov19 🤝

- [ ] As I can see from the previous review comment. You need to stick to the design given in the Figma template as much as possible. At the moment your popup doesn't replicate the design given.
- Your design
Your design
- Given design
given-design

- [ ] [Optional] You may also implement the button interactions of `hover` on your popup windows.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @maov19 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions

- [ ] You may always maintain the same pull request where changes were requested to enable other reviewers to perform more efficient reviews.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by maov19** [comment]
### Dear reviewer, please read before rejecting.
The stylelint error is caused because of the }; in line 1014 in the style.css file.
If this is deleted, the popups aren't displayed correctly. Please take a look at the screenshots below.
My PR was rejected +3 times because of this designing issue. 
Please test it and let me know if you know how to fix it. I'll appreciate your help. 

_PS: I found [this info](https://stackoverflow.com/questions/8748823/what-does-a-semicolon-outside-of-curly-braces-do/8748860), but deleting the expression doesn't fix the issue._

Thanks!

- Including the }: in line 1014
image

- Excluding the }: in line 1014
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @maov19 

# Invalid pull request 🔴 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But am sorry I can't review your project because the linters are not all passing. kindly fix this an send for review 

Optional suggestions
 You may always maintain the same pull request where changes were requested to enable other reviewers to perform more efficient reviews.
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me @1GORDON if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by maov19** [comment]
Hi @1GORDON ! The explanation for the linters issue is in the comment above. 
I can fix it, but that will cause a design issue. Can you please take a look at it?
Thanks!

**Comment by Shedrack-Sunday** [comment]
> Hi @1GORDON ! The explanation for the linters issue is in the comment above. I can fix it, but that will cause a design issue. Can you please take a look at it? Thanks!

Hello @maov19, if it causes a design issue, am sorry but you have to fix that. Your review can only be checked if your linters all pass. Besides, I don't think a linter fix will significantly alter a code base. Kindly follow the suggestion 👏

**Comment by maov19** [comment]
@1GORDON, the issue is fixed!
As [I explained](https://github.com/maov19/my_portfolio/pull/15#issuecomment-1416643472), this will cause a design issue. I'll appreciate if you let me know bejore rejecting it again. 
It took me some days and 6 reviews to find out what was going on.

Thanks!

File: app.js [hunk 2]
```diff
@@ -22,53 +22,54 @@ const portfolios = [
...
-    content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    image: './images/project4_desktop.png',
+    content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
     seelive: 'See live',
     seesource: 'See Source',
```
Comment (roshan-bajgain) [inline app.js:55]: - [ ] [Optional] I would suggest you please use the blurred background so the image and text don't overlap to each other and if you see the design then you will find blur background beside the popup window.
- [ ] Also please this is your last review so I am providing you optional kindly cover the image with the whole width.
### Your design
his


### Figma design 

figma

**Review by roshan-bajgain — APPROVED** [review]
Hi @maov19, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 2 above)
Comment (maov19) [reply app.js:55]: @roshan-bajgain thanks, I'll fix that during the weekend!

**Pull request merged**